v1/e2e_test.go

```diff
@@ -0,0 +1,1102 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/vdaas/vald-client-go/v1/agent/core"
+	"github.com/vdaas/vald-client-go/v1/payload"
+	"github.com/vdaas/vald-client-go/v1/vald"
+	"google.golang.org/grpc"
+)
+
+const addr = "localhost:8081"
+
+var (
+	conn *grpc.ClientConn
+
+	client      vald.Client
+	agentClient core.AgentClient
+
+	ctx context.Context
+
+	data []Datum
+
+	insertConfig = &payload.Insert_Config{
+		SkipStrictExistCheck: true,
+	}
+
+	searchNum    = 3
+	searchConfig = &payload.Search_Config{
+		Num:     uint32(searchNum),
+		Radius:  -1.0,
+		Epsilon: 0.1,
+		Timeout: 3000000000,
+	}
+
+	updateConfig = &payload.Update_Config{
+		SkipStrictExistCheck: true,
+	}
+
+	upsertConfig = &payload.Upsert_Config{
+		SkipStrictExistCheck: true,
+	}
+
+	removeConfig = &payload.Remove_Config{
+		SkipStrictExistCheck: true,
+	}
+)
+
+type Datum struct {
+	ID     string    `json:"id"`
+	Vector []float32 `json:"vector"`
+}
+
+func init() {
+	ctx = context.Background()
+
+	err := readJSON()
+	if err != nil {
+		fmt.Printf("error: %s", err)
+		os.Exit(1)
```

Comment:
[nits] 
we can use `log.Fatalf` instead of this code.
Because `fmt.Printf` and `os.Exit(1)` is same as `log.Fatal` 🤔 

https://cs.opensource.google/go/go/+/refs/tags/go1.16.6:src/log/log.go;l=333

```suggestion
		log.Fatalf("error: %s", err)
```